### PR TITLE
Add integration test for kibana notebooks

### PR DESCRIPTION
**Pull request by gaiksaya**
*Issue #, if available:*
P43421627

*Description of changes:*
- Add integration test for kibana notebooks
- Modify installation script for windows
- Modify windows build workflow to adapt to new folder structure and yaml parser
- Replace `windows-latest` with `windows-2019` in the windows workflow

*Test Results:*

- Tar: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/483881993
- Deb: https://github.com/opendistro-for-elasticsearch/opendistro-build/runs/1698177061?check_suite_focus=true
- RPM: https://github.com/opendistro-for-elasticsearch/opendistro-build/runs/1698447048?check_suite_focus=true
- Docker: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/483959095
- Windows: https://github.com/opendistro-for-elasticsearch/opendistro-build/runs/1731694564?check_suite_focus=true
https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/497304062

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/staging-build-deb.yml [hunk 1]
```diff
@@ -583,6 +583,83 @@ jobs:
...
+      - name: Set up AWS Cred
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Retrieve plugin tags 
+        run: |
+          echo "p_tag_kib_nb=$(release-tools/scripts/plugin_tag.sh opendistro-for-elasticsearch/kibana-notebooks)" >> $GITHUB_ENV
```
Comment (sreekarjami) [inline .github/workflows/staging-build-deb.yml:605]: As this is new implementation, please pick the tag version from Manifest.

Comment (gaiksaya) [reply .github/workflows/staging-build-deb.yml:605]: I have not tested with new implementation. May be we do a global replacement and test all at once. That will also include separating the IT altogether in new workflow

Comment (sreekarjami) [reply .github/workflows/staging-build-deb.yml:605]: Sure.

**Review by sreekarjami — APPROVED** [review]

**Review by peterzhuamazon — APPROVED** [review]

File: release-tools/scripts/setup_runners_service_windows.ps1 [hunk 2]
```diff
@@ -22,14 +22,16 @@
 
 # setup user parameters
 $SETUP_ACTION=$args[0]
-if (!$SETUP_ACTION) {
-  echo "Please enter 1 parameter: --es | --es-nosec | --kibana | --kibana-nosec"
+$OD_VERSION=$args[1]
+if (!$SETUP_ACTION -or !$OD_VERSION) {
+  echo "Please enter 2 parameters: --es | --es-nosec | --kibana | --kibana-nosec 1.12.0"
```
Comment (sreekarjami) [inline release-tools/scripts/setup_runners_service_windows.ps1:27]: Why are we hardcoding the version --kibana-nosec 1.12.0

File: release-tools/scripts/setup_runners_service_windows.ps1 [hunk 3]
```diff
@@ -22,14 +22,16 @@
...
+  echo "Example: $0 --es-nosec 1.12.0"
```
Comment (sreekarjami) [inline release-tools/scripts/setup_runners_service_windows.ps1:28]: Why are we hardcoding the version.

File: release-tools/scripts/setup_runners_service_windows.ps1 (hunk 2 above)
Comment (peterzhuamazon) [reply release-tools/scripts/setup_runners_service_windows.ps1:27]: That is the help message as an example, but can be changed to things like <ODFE Version> for more clarity.

File: release-tools/scripts/setup_runners_service_windows.ps1 (hunk 3 above)
Comment (peterzhuamazon) [reply release-tools/scripts/setup_runners_service_windows.ps1:28]: Same as above

File: release-tools/scripts/setup_runners_service_windows.ps1 (hunk 2 above)
Comment (sreekarjami) [reply release-tools/scripts/setup_runners_service_windows.ps1:27]: Can we have it generic like 1.x.x instead of a specific version.

Comment (gaiksaya) [reply release-tools/scripts/setup_runners_service_windows.ps1]: updated

File: release-tools/scripts/setup_runners_service_windows.ps1 (hunk 3 above)
Comment (gaiksaya) [reply release-tools/scripts/setup_runners_service_windows.ps1]: Updated

**Pull request merged**